libs/shared/base-abstract-classes/src/service/editors/single-row-editor.service.ts

```diff
@@ -86,19 +86,27 @@ export abstract class SingleRowEditorService<T extends TableRow> extends EditorS
 
   protected updateFormAfterReload() {
```

Comment:
Not related to the PR
```suggestion
  protected updateFormAfterReload(): void {
```